### PR TITLE
bpo-34282: Fix Enum._convert shadowing members named _convert

### PR DESCRIPTION
**Pull request by orlnub123**
I've combined the third solution with the second from the issue below, circumventing its problem.

https://bugs.python.org/issue34282

### CODE REVIEW

**Comment by orlnub123** [comment]
Is a news entry needed?

**Comment by orlnub123** [comment]
I've gone ahead and added a news entry.

File: Lib/enum.py [hunk 1]
```diff
@@ -424,6 +424,39 @@ def _create_(cls, class_name, names, *, module=None, qualname=None, type=None, s
 
         return enum_class
 
+    def _convert(cls, name, module, filter, source=None):
```
Comment (ethanfurman) [inline Lib/enum.py]: See notes in bpo issue.  Thanks for your help in this!

**Review by ethanfurman — CHANGES_REQUESTED** [review]

File: Lib/enum.py (hunk 1 above)
Comment (orlnub123) [reply Lib/enum.py]: Glad to help! I've got some questions about them in the issue.

**Comment by orlnub123** [comment]
I have made the requested changes; please review again.

**Comment by orlnub123** [comment]
I've merged the changes from #9034.

**Review by ethanfurman — CHANGES_REQUESTED** [review]
Looks good!  We just need one more test that verifies that usage of '_convert' in 3.8 raises a warning, and in 3.9+ raises an AttributeError.

Oh, and make sure you are in the contributors file.  :)

**Comment by orlnub123** [comment]
Thanks for the offer, I really appreciate it. I hope I've added myself to the right place!

I have made the requested changes; please review again.

**Review by ethanfurman — APPROVED** [review]

**Pull request merged**